khelo/lib/components/won_by_message_text.dart

```diff
@@ -26,21 +29,42 @@ class WonByMessageText extends StatelessWidget {
               AppTextStyle.subtitle1
                   .copyWith(color: context.colorScheme.textPrimary));
     }
-    return Text.rich(TextSpan(
-        text: matchResult!.teamName,
-        style: textStyle ??
-            AppTextStyle.subtitle2
-                .copyWith(color: context.colorScheme.textPrimary),
+
+    if (isTournament) {
+      return Column(
         children: [
-          TextSpan(
-              text: context.l10n.score_board_won_by_title,
-              style: textStyle ??
-                  AppTextStyle.body2
-                      .copyWith(color: context.colorScheme.textDisabled)),
-          TextSpan(
-            text: matchResult!.winType
-                .getString(context, matchResult!.difference),
+          Text(
+            '${matchResult!.teamName.initials(limit: 2)} Won',
+            style: textStyle ??
+                AppTextStyle.subtitle2
+                    .copyWith(color: context.colorScheme.textPrimary),
+          ),
```

Comment:
what is the main difference between the widget for tournament and not for tournament? also consider to localize string for "Won" and "by" texts.